package.json

```diff
@@ -134,6 +134,9 @@
     "@kbn/ambient-ui-types": "link:bazel-bin/packages/kbn-ambient-ui-types",
     "@kbn/analytics": "link:bazel-bin/packages/kbn-analytics",
     "@kbn/analytics-client": "link:bazel-bin/packages/analytics/client",
+    "@kbn/analytics-shippers-elastic-v3-common": "link:bazel-bin/packages/analytics/shippers/elastic_v3/common",
+    "@kbn/analytics-shippers-elastic-v3-server": "link:bazel-bin/packages/analytics/shippers/elastic_v3/server",
+    "@kbn/analytics-shippers-elastic-v3-ui": "link:bazel-bin/packages/analytics/shippers/elastic_v3/ui",
```

Comment:
How do you feel about using `-browser` for the suffix here? The planned "package type" for packages like this will be `shared-browser`.

Reply:
makes sense! I'll change it!